src/IslandoraUtils.php

```diff
@@ -755,4 +768,71 @@ protected function getParentsByEntityReference(ContentEntityInterface $entity, a
     return $parents;
   }
 
+  /**
+   * Deletes Media and all associated files.
+   *
+   * @param object $media
+   *   Ids of media to be deleted.
```

Comment:
From the look of the code, these are media entities, not IDs. Perhaps `object[]` and change the description to match?

Reply:
or `\Drupal\media\MediaInterface`?